test/fixtures/dialects/postgres/postgres_alter_schema.sql

```diff
@@ -0,0 +1,2 @@
+ALTER SCHEMA schema1 RENAME TO schema2;
+ALTER SCHEMA schema1 OWNER TO new_owner;
```

Comment:
Nit:

```suggestion
ALTER SCHEMA schema1 OWNER TO new_owner;

```

Reply:
To use git properly am I supposed to **_accept_** this suggestion somehow? or should I **_make_** this change?
Obviously it's extremely easy for me to add the new line... I just want to be sure that I'm using git correctly.

Reply:
Nevermind. I see how to commit your suggestion now.

Reply:
You can do either!

For simple changes like this you can use the `commit suggestion` button above.
To accept multiple suggestions, you can batch the acceptances and then commit in one go (quicker than accepting individually if lots of them).
Either of those will resolve this conversation automatically.
Note you'll need to pull these changes from GitHub to your local branch if you want to make any more changes locally.

However, if there's lots of other changes than can do offline and push. At which point these conversations will show as "outdated". you can then flip through all outdated ones and resolve them if happy they outdating did indeed resolve the conversation.